bioio/array_like_reader.py

```diff
@@ -110,13 +110,29 @@ class ArrayLikeReader(Reader):
     def _is_supported_image(  # type: ignore
         image: Union[List[MetaArrayLike], MetaArrayLike], *args: Any, **kwargs: Any
     ) -> bool:
+        errors = []
         if isinstance(image, list):
-            return all(
-                isinstance(scene, (np.ndarray, da.Array, xr.DataArray))
-                for scene in image
+            for scene in image:
+                if isinstance(scene, (np.ndarray, da.Array, xr.DataArray)):
+                    continue
+                else:
+                    errors.append(f"Unsupported scene type: {type(scene)}. ")
+        else:
+            if isinstance(image, (np.ndarray, da.Array, xr.DataArray)):
+                return True
+            else:
+                errors.append(f"Unsupported image type: {type(image)}. ")
+        if errors:
+            errors.append(
+                "ArrayLikeReader supported types are numpy ndarray, dask Array,"
+                + "or xarray DataArray."
             )
-
-        return isinstance(image, (np.ndarray, da.Array, xr.DataArray))
+            raise exceptions.UnsupportedFileFormatError(
+                "ArrayLikeReader", ", ".join(errors)
+            )
```

Comment:
I dont think that we need all of this if else complexity. I think the original `if isinstance` was really just there so we could use `all()`. Could we do something like 
```
SUPPORTED = (np.ndarray, da.Array, xr.DataArray)
imgs = image if isinstance(image, list) else [image]
    for img in imgs:
        if not isinstance(img, SUPPORTED):
            raise exceptions.UnsupportedFileFormatError()
Return True
```